packages/ilmomasiina-backend/src/authentication/adminAuthSession.ts

```diff
@@ -14,7 +14,7 @@ export interface AdminTokenData {
 
 export default class AdminAuthSession {
   /** Session lifetime in seconds */
-  static TTL = config.nodeEnv === 'development' ? 365 * 24 * 60 * 60 : 10 * 60;
+  static TTL = config.nodeEnv === 'development' ? 365 * 24 * 60 * 60 : 3 * 60 * 60;
```

Comment:
```suggestion
  static TTL = config.nodeEnv === 'development' ? 365 * 24 * 60 * 60 : 1 * 60 * 60;
```
Instead of 3 hours, have it at 1 is reasonable enough 🤔